google/ggrc-core
# GGRC-6390 Remove hide link from label tag

# Issue description

Extract 'hide' link from 'label' tag in create and edit forms of all objects

# Steps to test the changes
1. Open any Create or Edit form, check where 'hide' link of form elements is located.
2. Inspect the 'hide' link elements in Dev Tools.
3. The 'hide' link elements should not be inside 'label' tag associated with the field.

# Solution description
1. New style attribute 'display' is set to 'inline-block' for 'label' tag of a 'modal' class element. The change is done in src/ggrc-client/styles/modules/_modal.scss.
2. Then modal pages of all objects are analysed to see if there are any hide links present in 'label' tags.
3. All such 'hide' links are removed from 'label' tag and placed below the 'label' tag.

# Sanity checklist




# PR Review checklist
